<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIFavoriteHubEditor</class>
 <widget class="QDialog" name="UIFavoriteHubEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Hub</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox_AUTOCONNECT">
          <property name="text">
           <string>Autoconnect on startup</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_DISABLECHAT">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Disable chat</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="LineEdit" name="lineEdit_NAME">
        <property name="toolTip">
         <string>Favorite hub name.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="LineEdit" name="lineEdit_ADDRESS">
        <property name="toolTip">
         <string>IP address or DNS name of hub.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="LineEdit" name="lineEdit_DESC">
        <property name="toolTip">
         <string>This description will be shown in the list of favorite hubs.</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Encoding</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBox_ENC"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Mode</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboBox_MODE">
        <item>
         <property name="text">
          <string>Default</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Active</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Passive</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="checkBox_CID">
        <property name="text">
         <string>Client ID</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="comboBox_CID">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="checkBox_IP">
        <property name="text">
         <string>External IP</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="LineEdit" name="lineEdit_IP">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="inputMask">
         <string>000.000.000.000; </string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QCheckBox" name="checkBox_USEINTERNET">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Auto update from Internet</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Minimum search interval (seconds)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_MINSEARCH_INTERVAL">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1200</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Identification</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="margin">
       <number>1</number>
      </property>
      <item row="1" column="1">
       <widget class="LineEdit" name="lineEdit_NICK">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>This nickname, or handle, will identify you when you join a hub.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="LineEdit" name="lineEdit_PASSWORD">
        <property name="toolTip">
         <string>If hub requires password, define it here.</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="LineEdit" name="lineEdit_USERDESC">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>This description will be shown in the user list.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_NICK">
        <property name="text">
         <string>Nick</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBox_USERDESC">
        <property name="text">
         <string>User description</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>LineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UIFavoriteHubEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UIFavoriteHubEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_NICK</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_NICK</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_USERDESC</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_USERDESC</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_CID</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox_CID</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_IP</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_IP</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_IP</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBox_USEINTERNET</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
